docs/CSBR.md

```diff
@@ -166,7 +171,7 @@ Capitalized Terms are as defined in the Baseline Requirements or the EV SSL Guid
 
 **Signature**: An encrypted electronic data file which is attached to or logically associated with other electronic data and which (i) identifies and is uniquely linked to the signatory of the electronic data, (ii) is created using means that the signatory can maintain under its sole control, and (iii) is linked in a way so as to make any subsequent changes that have been made to the electronic data detectable.
 
-**Signing Service**: An organization that signs Code on behalf of a Subscriber using a Private Key associated with a Code Signing Certificate.
+**Signing Service**: An organization that signs Code on behalf of a Subscriber using a Private Key associated with the Subscriber's Code Signing Certificate.

```

Comment:
I believe we have already described the use case where the user may only be sending a "hash-to-be-signed" and not the entire Code, so we might need to take that into consideration. We should also highlight that this is a "remote" service.

Here is my proposal:

**Signing Service**: An organization that remotely signs Code or data associated with Code, on behalf of a Subscriber using a Private Key associated with the Subscriber's Code Signing Certificate.

Reply:
Is is not always a hash? Either the Signing Service creates the hash or receives a hash. Or do I have that wrong?

Reply:
The definition of "Code" is :

> **Code:** A contiguous set of bits that has been or can be digitally signed with a Private Key that corresponds to a Code Signing Certificate.

So, a Signing Service either receives the Code and signs it (by creating the hash, etc, etc) or receives a hash from the Subscriber and signs that hash.

Reply:
I don't think changes are needed here.  It has been historically understood that creating an intermediate hash, and then signing the hash, is how signing works.

Unless we're going to start adding requirements about inspecting the code to be signed (which is actually a legitimate conversation topic), this detail doesn't matter.  If a signing service receives a hash and signs it, it has "signed Code", despite never having seen it.  In a model where the signature simply exists to associate an identity with the Code at the signer's request, without any inspection of the Code, this works just fine.  And, for better or worse, that's the current signing model in this ecosystem.